arm-apimanagement/2016-07-07/swagger/apimmanagement.json

```diff
@@ -6321,6 +6574,73 @@
         "type"
       ],
       "description": "Parameters supplied to the Update Logger operation."
+    },    
+    "BackendGetContract": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Uniquely identifies the backend within the current API Management service instance. The value is a valid relative URL in the format of /backends/{backendId} where {backendId} is a backend identifier.",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackendParameters"
+        }
+      ],
+      "required": [
+        "host",
+        "skipCertificateChainValidation"
+      ],
+      "description": "Backend."
+    },
+    "BackendPaged": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BackendGetContract"
+          },
+          "description": "Backend values."
+        },
+        "count": {
```

Comment:
count will be ignored. Is this a standard thing to have? I have only seen value and nextLink property for pageable oprerations.